### PR TITLE
Added animation

### LINKED ISSUE #250: Feature Request: Adding Animation
* **Description:** Describe the feature you would like to see added to CodeHarborHub. Explain how this feature would benefit users.

- Your landing page already looks fantastic, but incorporating animations could make it even more captivating. With my experience in this area, I'm eager to contribute to this enhancement.
- If you could assign me this task under GSSOC24, I would be thrilled to work on it.

* **Use Case:** Provide a specific example of how you would use this feature. 

https://github.com/CodeHarborHub/codeharborhub/assets/115486960/35087e59-8c9c-44c9-b5eb-019d7f47d035

* **Similar Features:** (Optional) If there are other platforms that offer a similar feature, mention them and explain how your proposed feature would be different or better.

### PR DESCRIPTION
**Pull request by harshmishra19**
## Related Issue

This PR fixes #250 

## Description

I have added the animation

## Type of PR

- [ ] Bug fix
- [ X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

https://github.com/CodeHarborHub/codeharborhub/assets/115486960/55cc416e-2501-458c-bf66-4e1ec8f2c87a

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [ x] I have performed a self-review of my code.
- [ x] I have read and followed the Contribution Guidelines.
- [ x] I have tested the changes thoroughly before submitting this pull request.
- [ x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x ] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: src/components/HomePage/vanilla-tilt.js [hunk 1]
```diff
@@ -0,0 +1,501 @@
+var VanillaTilt = (function () {
```
Comment (Ajay-Dhangar) [inline src/components/HomePage/vanilla-tilt.js:1]: You can try `const` and `let` not good practice for using in js `var` keyword

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
please checkout and apply best practice rules

**So, Now resolve all**

- [ ] [Unexpected var, use let or const instead](https://app.deepsource.com/gh/CodeHarborHub/codeharborhub/run/4f9d43dc-1e65-4a27-bce9-82e75d32baa0/javascript/JS-0239?listindex=0)
- [ ] [Should not have unused variables](https://app.deepsource.com/gh/CodeHarborHub/codeharborhub/run/4f9d43dc-1e65-4a27-bce9-82e75d32baa0/javascript/JS-0128?listindex=1)
- [ ] [Use shorthand property syntax for object literals](https://app.deepsource.com/gh/CodeHarborHub/codeharborhub/run/4f9d43dc-1e65-4a27-bce9-82e75d32baa0/javascript/JS-0240?listindex=2)
- [ ][ Logical operator can be refactored to optional chain](https://app.deepsource.com/gh/CodeHarborHub/codeharborhub/run/4f9d43dc-1e65-4a27-bce9-82e75d32baa0/javascript/JS-W1044?listindex=12)
- [ ] [more](https://app.deepsource.com/gh/CodeHarborHub/codeharborhub/run/4f9d43dc-1e65-4a27-bce9-82e75d32baa0/javascript?page=1)

**Comment by Ajay-Dhangar** [comment]
Good work, only debug some code and make good code for us.

**Review by Ajay-Dhangar — APPROVED** [review]
Now, I merged in this time and next time you try to follow best practice rules.

**Comment by harshmishra19** [comment]
YESS!!!!

**Pull request merged**